chemreg/compound/tests/test_models.py

```diff
@@ -45,6 +51,28 @@ def test_definedcompound_attr():
     assert inchikey.max_length == 29
 
 
+@pytest.mark.django_db(transaction=True)
```

Comment:
It would be nice to avoid the DB test scenarios as much as possible early on and only rely on them when absolutely necessary. This will make testing much easier down the road as the test suite grows. We can reduce the entire test suite run time from minutes to seconds.

Below are some alternative `assert`s that do not rely on the database.